src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -197,49 +218,69 @@ class DetailsSidebar extends React.PureComponent<Props, State> {
     }
 
     /**
-     * Sucess callback for classification change
+     * File classification fetch success callback.
      *
-     * @private
+     * @param {ClassificationInfo} classificationInfo - Info about the file's classification
      * @return {void}
      */
-    classifiationChangeSuccessCallback = (file: BoxItem) => {
+    fetchClassificationSuccessCallback = (classificationInfo: ClassificationInfo): void => {
         this.setState({
-            file,
-            isLoading: false,
+            classificationInfo,
+            classificationError: undefined,
+            isLoadingClassification: false,
         });
     };
 
     /**
-     * Error callback for classification change
+     * Handles a failed file classification fetch
      *
      * @private
+     * @param {ElementsXhrError} error - API error
      * @return {void}
      */
-    classifiationChangeErrorCallback = () => {
+    fetchClassificationErrorCallback = (error: ElementsXhrError): void => {
+        let classificationError;
+
+        if (getProp(error, 'status') === HTTP_STATUS_CODE_FORBIDDEN) {
```

Comment:
Why do we need to show error? If the feature is turned off, then hasClassification should be false from the webapp level.

Reply:
You mean the unauthorized error? Will remove, since that makes the most sense

Reply:
Ah, but in the case that a developer outside of webapp runs into this, though...